charts/kyverno/Chart.yaml

```diff
@@ -34,9 +34,7 @@ annotations:
   # valid kinds are: added, changed, deprecated, removed, fixed and security
   artifacthub.io/changes: |
     - kind: fixed
-      description: switch to post-delete helm hook to clean up Kyverno configmap
-    - kind: fixed
-      description: modify config.webhooks to be as a single object instead of a list of webhooks.
+      description: use clusterephemeralreports/ephemeralreports instead of clusterbackgroundscanreports/backgroundscanreports in validate.yaml
```

Comment:
I would describe "what" issue is fixed instead of "how" the issue is fixed🤔

Reply:
Done.